docs/foundations/icons.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Icons
+---
+
+## Intro
+
+SF DesSys icons are visual representations of ideas and actions. These icons allow our users to quickly see interface intent at a glance and take part in establishing hierarchy.
+
+Icons also assist with accessibility in scenarios of low literacy or English proficiency.
+
+## Principles
+
+Clarity
+
+Culturally aware
+
+*Add potential principles here to be fleshed out
+
+## Design Guidelines
+
+### Interface Icons vs. Picture Icons
+
+The San Francisco Design System utilizes two types of images to assist users; interface icons and Picture icons.
+
+Interface icons, or just icons, are generally used to convey an action or intent and are not illustrative or decorative.
+
+Picture icons serve some of the same purposes as the interface icons but in a more expressive way.
+
+### Deciding on a new icon
+
+Before creating a new icon, pause and look for an icon in the SF DesSys icon library that clearly conveys your message; there’s no reason to bloat the icon library for the sake of having variety.
+
+<img width="250" alt="A calendar icon being used with the word Events" src="https://user-images.githubusercontent.com/957314/151463326-d263db7a-452a-4cfc-9d10-033dc09212f0.png">
+
+<img width="250" alt="A calendar icon being used with a form field" src="https://user-images.githubusercontent.com/957314/151463860-b946082b-761e-4a4c-92d8-a30b592c6ba5.png">
+
+Once you’ve decided that there is a need for a new icon, you have to decide on the best way to represent the concept. In most cases, there is a representation of that idea that already exists.
+
+<figure>
+  <img width="612" alt="Examples of delete icons" src="https://user-images.githubusercontent.com/957314/151464294-ac829556-2d12-49bb-b48d-14098d5dce13.png">
+  <figcaption align="center">A search for delete icon will generally display a trashcan or an x mark</b></figcaption>
+</figure>
+
+
+### Creating a new icon
+
+Great! You’ve decided there is a need for this icon, and you know exactly what it should be. We’ve created guidelines to help your new icon feel like a cohesive member of the San Francisco Design System.
+
+#### Interface icons
+
+##### Workspace
```

Comment:
I think this is the first time we've used an `h6` on the page? it's pretty tiny so I fear it won't pass visual accessibility muster, which probably means we should change it to something larger (I'm open to suggestions), _but_, it also raises whether we should implement guidelines for/against `h6` usage

Reply:
I changed it up to remove the need for an h6. h5 may be as deep as we will want to go.

Reply:
hm I'm still seeing `h6`'s. we 'behead' heading elements by 1, so `h1` becomes `h2` becomes `h3` and so on. although we might change this after christina made some accessibility recs, so maybe we can worry about this later